hieradata/cluster/corels.yaml

```diff
@@ -0,0 +1,2 @@
+---
+firewall::ensure: "stopped"
```

Comment:
This should be pushed to the entire site.

Reply:
Agreed. I'll open a PR to move come values up from the cluster level.

Reply:
https://github.com/lsst-it/lsst-itconf/pull/41